<launch>
	<node name="aruco_analyzer" pkg="aruco_analyzer" type="aruco_detection_node" output="screen" respawn="false" clear_params="true"> 
		<rosparam command="load" file="$(find aruco_analyzer)/config/config.yaml" param="config"/>
		<param name="transport_hint" value="/image_raw" />
		<param name="camera_ns" value="/cameras" />
	</node>
</launch>
